nautobot/docs/additional-features/jobs.md

```diff
@@ -326,6 +326,9 @@ Similar to `ObjectVar`, but allows for the selection of multiple objects.
 
 An uploaded file. Note that uploaded files are present in memory only for the duration of the job's execution: They will not be automatically saved for future use. The job is responsible for writing file contents to disk where necessary.
 
+!!! note
+    Keep in mind that the maximum size is defined by the FILE_UPLOAD_MAX_MEMORY_SIZE from Django settings. By default, it uses 2621440 (i.e. 2.5 MB).
```

Comment:
It would be good to also document this under the `Django Configuration Settings` section in `nautobot/docs/user-guide/administration/configuration/optional-settings.md`, I think.

Reply:
yes, I had an old version of the repo